marketpulse/main/templates/activities.html

```diff
@@ -23,8 +23,9 @@ <h4 class="text-center">
 
   <div class="row">
     <div class="col-md-12 text-right">
-      <a href="{{ url('main:user_fxosprice', user.pk) }}" class="view-all">View your contributions</a> &bull;
-      <a href="{{ url('main:all_fxosprice') }}" class="view-all">View all contributions</a>
+      <a href="{{ url('main:user_contributions') }}"
+         class="view-all">View your contributions</a> &bull;
```

Comment:
Do we really need to split this line? It doesn't look much bigger than previous/next ones.